resources/assets/fontello/demo.html

```diff
@@ -0,0 +1,321 @@
+<!DOCTYPE html>
```

Comment:
maybe remove the redundant demo

Reply:
Yes. Right now I use it for whenever I want to add an icon, but removing it is probably a good idea. I'll also move the License to the root